style.css

```diff
@@ -68,5 +133,58 @@ aside {
 }
 
 .margin-padding-list {
-    padding-left: 16px;
+    padding-left: 40px;
+}
+
+@media (max-width: 1700px) {
+    .main-columns {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .instructions {
+        width: 90%;
+        margin-top: 150px;
+    }
+
+    .ingredients {
+        width: 95%;
+        justify-content: flex-end;
+    }
+}
+
+@media (max-width: 1700px) {
```

Comment:
Don't use `@media (max-width: 1700px)` two times. Instead you can fit everything in the same block.